### PR TITLE
dlinknet uses mindcv resnet backbone

### PR DESCRIPTION
**Pull request by zangqx-2024**
1、dlinknet uses mindcv resnet backbone
2、epoch_num to epoch_size

### CODE REVIEW

File: official/cv/dlinknet/README_CN.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ D-LinkNet模型基于LinkNet架构构建。实现方式见论文[D-LinkNet: Link
  ```
 
 ## 预训练权重
-- [Resnet34 Imagenet Checkpoint](https://download-mindspore.osinfra.cn/toolkits/models/dlinknet/resnet34_ascend_v130_imagenet2012_official_cv_bs256_top1acc73.83_top5acc91.61.ckpt)
-- [Resnet50 Imagenet Checkpoint](https://download-mindspore.osinfra.cn/toolkits/models/dlinknet/resnet50_ascend_v1100_imagenet2012_official_cv_top1acc76.97_top5acc93.44.ckpt)
+- [Resnet34 Imagenet Checkpoint](https://download-mindspore.osinfra.cn/toolkits/models/dlinknet/resnet34-f297d27e.ckpt)
```
Comment (CaitinZhao) [inline official/cv/dlinknet/README_CN.md:18]: 直接放mindcv的link

File: official/cv/dlinknet/models/dlinknet.py [hunk 2]
```diff
@@ -1,12 +1,75 @@
...
+from mindcv import create_model
 
 from mindspore import nn, load_checkpoint, load_param_into_net
 from mindspore.common.tensor import Tensor
 from mindspore.mint.nn.functional import relu, batch_norm
 from mindspore.mint import sigmoid
-from models.resnet import resnet34, resnet50, kaiming_uniform
 from config import parse_args
 
+
```
Comment (CaitinZhao) [inline official/cv/dlinknet/models/dlinknet.py:12]: 这些可以不加，加载预训练权重的话这部分没有用

File: official/cv/dlinknet/requirements.txt [hunk 3]
```diff
@@ -2,3 +2,4 @@ opencv-python >= 4.1.2.30
 numpy
 pyyaml
 scikit-learn
+mindcv
```
Comment (CaitinZhao) [inline official/cv/dlinknet/requirements.txt:5]: 加一下版本

File: official/cv/dlinknet/README_CN.md (hunk 1 above)
Comment (zangqx-2024) [reply official/cv/dlinknet/README_CN.md]: 已调整

File: official/cv/dlinknet/requirements.txt (hunk 3 above)
Comment (zangqx-2024) [reply official/cv/dlinknet/requirements.txt]: 已增加

**Review by CaitinZhao — APPROVED** [review]

**Review by SamitHuang — APPROVED** [review]
